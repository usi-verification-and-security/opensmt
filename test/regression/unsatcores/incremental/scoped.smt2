(set-option :produce-unsat-cores true)
(set-logic QF_LIA)
(declare-fun o () Int)
(declare-fun i () Bool)
(assert (or (not i) (< o 0)))
(push 1)
(assert (! i :named c0))
(check-sat)
(pop 1)
(declare-fun __ () Bool)
(assert (or (>= o 0) (not __)))
(assert (! __ :named c15))
(assert (! i :named c17))
(check-sat)
(get-unsat-core)
