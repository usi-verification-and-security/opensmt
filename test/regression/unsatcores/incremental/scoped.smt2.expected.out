sat
unsat
(
c17
c15
)
